.github/workflows/release-build.yml

```diff
@@ -125,7 +144,9 @@ jobs:
     steps:
     - name: Checkout event ref
       uses: actions/checkout@v2
-      if: github.event.action == 'published'
+      with:
+        ref: ${{ github.event.client_payload.commit }}
+      if: github.event.action == 'published' || github.event.action == 'helics-release-build'
```

Comment:
This one doesn't have conditions like some of the others,  just want to make sure that is intentional?

Reply:
It's above the matrix section -- https://github.com/GMLC-TDC/HELICS/pull/1123/files/fd29bb2185f602ccb6567c6d084aaa80002486dc#diff-01667485dced5bf57d8837edbffc6507R117

Reply:
https://github.com/GMLC-TDC/HELICS/blob/fd29bb2185f602ccb6567c6d084aaa80002486dc/.github/workflows/release-build.yml#L117

Reply:
The checkout steps were changed so that it would use this one for published/event builds, and the fixed 'develop' branch checkout for the scheduled nightly build.